crates/bevy_input/src/keyboard.rs

```diff
@@ -88,7 +88,7 @@ use bevy_reflect::{ReflectDeserialize, ReflectSerialize};
 ///
 /// The event is consumed inside of the [`keyboard_input_system`]
 /// to update the [`ButtonInput<KeyCode>`](ButtonInput<KeyCode>) resource.
-#[derive(Event, Debug, Clone, PartialEq, Eq)]
+#[derive(Event, Debug, Clone, PartialEq, Eq, Hash)]
 #[cfg_attr(feature = "bevy_reflect", derive(Reflect), reflect(Debug, PartialEq))]
```

Comment:
Should `reflect(Hash, ...)` be added as well? I haven't used the reflection system enough to know when and when not to apply it.

Reply:
There's a bunch of code examples in bevy that implements `reflect(Hash)`, so I'd guess yes here.